src/main/java/org/opensearch/knn/plugin/script/KNNAllowlistExtension.java

```diff
@@ -16,9 +16,9 @@
 import java.util.List;
 import java.util.Map;
 
-public class KNNWhitelistExtension implements PainlessExtension {
+public class KNNAllowlistExtension implements PainlessExtension {
 
-    private static final Whitelist ALLOW_LIST = WhitelistLoader.loadFromResourceFiles(KNNWhitelistExtension.class, "knn_whitelist.txt");
+    private static final Whitelist ALLOW_LIST = WhitelistLoader.loadFromResourceFiles(KNNAllowlistExtension.class, "knn_allowlist.txt");
 
     @Override
     public Map<ScriptContext<?>, List<Whitelist>> getContextWhitelists() {
```

Comment:
Do we have a plan to change this line as well?

Reply:
As mentioned in my actual PR. The nomenclature changes from whitelist to allowlist are not being supported by opensearch core in 2.x version and are only supported from version 3.0. We are using a couple of opensearch core whitelist packages. We will update these lines of code after we increment our version to 3.0